script.py

```diff
@@ -0,0 +1,270 @@
+"""To use this script, install the provided requirements.txt file and
+supply a secrets.py file with the required parameters listed below.
+
+JSON records will be written into the specified file and HTML reports
+will be written into an outputs filter under the provided directory.
+"""
+
+
+from datetime import datetime, timedelta
+from jinja2 import Template
+from ua_parser import user_agent_parser
+
+import collections
+import json
+import os
+import requests
+import secrets
+import smtplib
+
+
+# This is the absolute path to the directory where you'd like
+# the JSON and HTML files to be written.
+# The template HTML file should also be placed in this
+# directory.
+DIR = secrets.DIR if hasattr(secrets, "DIR") else os.getcwd()
```

Comment:
I think we might need to do something referencing the actual location of this file using `__file__`, else this will not have the expected value if run from e.g. cron. c.f. https://stackoverflow.com/questions/14886715/wrong-path-results-the-python-program-runs-using-crontab